modules/claims_api/app/controllers/claims_api/metadata_controller.rb

```diff
@@ -6,14 +6,21 @@ class MetadataController < ::ApplicationController
     skip_after_action :set_csrf_header
     skip_before_action(:authenticate)
 
-    def index
+    def index # rubocop:disable Metrics/MethodLength
       render json: {
         meta: {
           versions: [
             {
-              version: '1.0.0',
+              version: '2.0.0',
               internal_only: false,
               status: VERSION_STATUS[:current],
+              path: '/services/claims/docs/v2/api',
+              healthcheck: '/services/claims/v2/healthcheck'
```

Comment:
lets make this be `/services/benefits/docs/v2/api` and `/services/benefits/v2/healthcheck`
easiest way to do this that ive found can be seen here:
https://github.com/department-of-veterans-affairs/vets-api/blob/36672a7cc8d6e8247cf597034263a1c45cd4a55e/config/routes.rb#L354

Reply:
Changed [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6845/commits/21b6bf1d705a060801414ee164315acd8de4a636).